src/js/components/LeftPane.js

```diff
@@ -1,38 +1,123 @@
+import Layout from "../state/Layout"
+import BookIcon from "../icons/BookSvgIcon"
+import styled from "styled-components"
+import usePopupMenu from "./hooks/usePopupMenu"
+import DropdownArrow from "../icons/DropdownArrow"
 
-export function LeftPane() {
-  let [showCollapse, setShowCollapse] = useState(true)
-  let view = useSelector(View.getInvestigationView)
-  let isOpen = useSelector(View.getLeftSidebarIsOpen)
-  let width = useSelector(View.getLeftSidebarWidth)
-  let dispatch = useDispatch()
+const Arrow = (props) => {
+  return (
+    <svg className={props.className} onClick={props.onClick} viewBox="0 0 8 8">
+      <polygon points="0 0 8 4 0 8" />
+    </svg>
+  )
+}
 
-  function onDrag(e: MouseEvent) {
-    const width = e.clientX
-    const max = global.innerWidth
-    dispatch(View.setLeftSidebarWidth(Math.min(width, max)))
+const SectionHeader = styled.div`
+  display: flex;
+  flex-direction: row;
+  background-color: ${(props) => props.theme.colors.coconut};
+  height: 32px;
+  align-items: center;
+  justify-content: flex-start;
+  border-top: 1px solid ${(props) => props.theme.colors.cloudy};
+  border-bottom: 1px solid ${(props) => props.theme.colors.cloudy};
+  margin-bottom: 10px;
+`
+
+const IconWrapper = styled.div`
+  svg {
+    width: 14px;
+    fill: ${(props) => props.theme.colors.aqua};
+    margin: 0 8px;
+  }
+`
+
+const Title = styled.label`
+  ${(props) => props.theme.typography.headingSection}
+`
+
+const StyledArrow = styled(Arrow)`
+  fill: ${(props) => props.theme.colors.lead};
+  width: 12px;
+  margin-left: 12px;
+  transform: ${(props) => (props.show ? `rotate(90deg)` : "")};
+  transition: transform 150ms;
+  cursor: pointer;
+`
+
+const ViewSelectWrapper = styled.div`
+  display: flex;
+  margin-left: auto;
+  margin-right: 15px;
+  cursor: pointer;
+`
+
+const StyledViewSelect = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  text-transform: capitalize;
+  color: ${(props) => props.theme.colors.slate};
+  ${(props) => props.theme.typography.labelMedium}
+
+  svg {
+    stroke: ${(props) => props.theme.colors.slate};
+    margin-left: 5px;
   }
+`
+
+type DropdownHeaderProps = {
+  icon: Node,
+  title: string,
+  viewSelect?: Node,
+  children: *
+}
+
+const DropdownHeader = ({
+  icon,
+  title,
+  viewSelect,
+  children
+}: DropdownHeaderProps) => {
+  const [show, setShow] = useState(true)
 
-  function onViewChange(name) {
-    dispatch(View.setInvestigationView(name))
+  const onClick = () => {
+    setShow(!show)
   }
 
-  function onClearAll() {
-    globalDispatch(Investigation.clearInvestigation())
+  return (
+    <section>
+      <SectionHeader>
+        <StyledArrow onClick={onClick} show={show} />
+        <IconWrapper>{icon}</IconWrapper>
+        <Title>{title}</Title>
+        <ViewSelectWrapper>{viewSelect}</ViewSelectWrapper>
+      </SectionHeader>
+      {show && children}
+    </section>
+  )
+}
+
+export function LeftPane() {
```

Comment:
I was just thinking last week that this last thing in the file is the thing that we export and all the private functions are above it. That way I always know I can scroll down to find the component that this file exports. Having the exported function end up in the middle always makes it hard to find. What do you think about that convention going forward. 

I often put private functions at the bottom of files since that's what I did in C++ and in ruby classes. But the convention above makes more sense to me in javascript. What do you think?